blog-posts/greenit-audit/greenit-audit.md

```diff
@@ -0,0 +1,51 @@
+---
+published: true
+title: 'You thought serverless was green? Here is how to really make it sustainable! 🌱'
+cover_image: 'http://www.nasa.gov/sites/default/files/thumbnails/image/as17-148-22727_lrg.jpg'
+description: 'Description of the article'
+tags: greenit, aws, sustainability, serverless
+series:
+canonical_url:
+---
+
+**Serverless is an amazing way to do cloud computing.** It removes the hassle of managing the infrastructure of your app and lets you focus on what really matters: delivering value.
+
+Moreover, in regular cloud computing, **lots of resources are idle**. Servers are often running without any task to do. People don’t optimize the capacity of their servers relative to the workload they really have. With serverless, cloud providers have a way to **share resources between clients and optimize their use**. Your serverless app will scale easily and will only be running when your users need it. For instance, AWS will launch Lambda functions instances and run their code only when required.
+
+![Lambda lifecycle](./assets/lambda-lifecycle.png 'Lambda lifecycle')
+
+Because of that, **serverless is often considered greener than traditional applications in the cloud**.
+
+## Is your serverless application perfectly sustainable?
+
+Going serverless is considered a sustainable solution to develop applications. **This is not a reason to be complacent**. You can certainly take action to improve the sustainability of your application.
```

Comment:
You can however implement by mistake anti-patterns that will impact negatively your actual resource consumptions.